src/client/Client.js

```diff
@@ -13,59 +14,66 @@ export default class DiscordClient {
 
-        this._client.on('message', (message) => {
-            console.log("> Client: on message");
-            const { member, content, guild } = message;
+        /** @var {Message} message */
+        this._client.on('message', (message) => this._handleMessage(message));
+    }
 
-            console.log(`Starts with ${process.env.COMMAND_PREFIX}? ${content.startsWith(process.env.COMMAND_PREFIX)}`);
-            if (!content.startsWith(process.env.COMMAND_PREFIX)) return;
-            
-            // Split on any number of spaces after the prefix
-            const args = content.slice(process.env.COMMAND_PREFIX.length).trim().split(/[ ]+/);
-            // Command typed by user
-            const userCommand = args.shift();
+    /**
+     * @param {string} token
+     * @return {Promise<string>}
+     */
+    login(token) {
+        return this._client.login(token);
+    }
 
-            console.log(`User command: ${userCommand}`);
+    /**
+     * @param {Message} message
+     * @return {void}
+     */
+    _handleMessage(message) {
+        const { content } = message;
+
+        if (!content.startsWith(process.env.COMMAND_PREFIX)) {
+            return;
+        }
 
-            // const command = `${process.env.PREFIX}${alias.toLowerCase()}`;
+        // Split on any number of spaces after the prefix
+        const args = content.slice(process.env.COMMAND_PREFIX.length).trim().split(/[ ]+/);
+        // Command typed by user
+        const userCommand = args.shift();
 
+        // const command = `${process.env.PREFIX}${alias.toLowerCase()}`;
 
-            const avaliableCommands = this._commandHandler.getCommands(message);
+        const avaliableCommands = this._commandHandler.getCommands(message);
 
-            if (avaliableCommands.has(userCommand)) {
-                const selectedCommand = avaliableCommands.get(userCommand);
-                const firstArgument = args[0];
+        if (avaliableCommands.has(userCommand)) {
+            const selectedCommand = avaliableCommands.get(userCommand);
+            const commandActionName = args[0];
 
-                if (0 < args.length) {
-                    if (typeof selectedCommand[firstArgument] === 'function') {
-                        // Delete function argument
-                        args.shift();
-                        selectedCommand[firstArgument](args.length > 0 ? {message, args} : {message});
-                        return;
-                    }
-                    selectedCommand.execute({message, args});
+            if (0 < args.length) {
+                if (typeof selectedCommand[commandActionName] === 'function') {
+                    // Remove action name from function arguments
+                    args.shift();
+
+                    selectedCommand[commandActionName](message, args);
                     return;
                 }
-                selectedCommand.execute({message});
+                selectedCommand.execute(message, args);
```

Comment:
Please remove 73 & 74 -  Add args in line 76: always pass args in command.execute function call.